presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -836,11 +836,12 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
             }
 
             external = true;
-            targetPath = getExternalPath(new HdfsContext(session, schemaName, tableName), externalLocation);
+            targetPath = new Path(externalLocation);
```

Comment:
Previously it was guarded with 
```
...
throw new PrestoException(INVALID_TABLE_PROPERTY, "External location is not a valid file system URI: " + location, e);
```

maybe extract a method like
```
private static Path getExternalLocationAsPath(String location) {
 ...
```

that would encapsulate the trycacth